docs/backend/pusher.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Pusher
+---
+
+# Pusher - Real-Time Notification
+
+Pusher is third party service being used to power the chat system. When both DEV.to members follow each other they have the ability to use chat and converse with each other.
```

Comment:
I'd remove `When both DEV.to members follow each other they have the ability to use chat and converse with each other.` mostly because it's not strictly true anymore. A user can activate open inboxes and get messaged by anyone, and also you have the group chats. I'd simplify just saying that Pusher powers the chat, unless you want to explain all the various cases.